src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -856,21 +856,20 @@ void FlexPA::prepPoint_pin_checkPoint_planar(
   auto layer = getDesign()->getTech()->getLayer(ap->getLayerNum());
   auto ps = std::make_unique<frPathSeg>();
   auto style = layer->getDefaultSegStyle();
+  const bool vert_dir = (dir == frDirEnum::S || dir == frDirEnum::N);
+  const bool horz_dir = (dir == frDirEnum::W || dir == frDirEnum::E);
```

Comment:
One is sufficient and implies the other. vert_dir=!horz_dir

Reply:
Ok, I was worried here that because dir can assume U (up) and D (down) maybe just setting vert_dir != horz_dir was not enough. Although the function name imples **planar** access (this is kept in the function name changes) I was worried for such corner cases

Reply:
When in doubt, look for the function usage in the code. You can see that dir is never passed as U or D. So it's safe to assume that dir is always planar.